### DISCUSSION TITLE
Video2Text: How to get text from any YouTube video

### DISCUSSION THREAD

**Discussion by XamHans** [openai/whisper#1600] [Show and tell]
This app converts any YouTube video to text

I played around with whisper some months ago and it resulted in this project.

Basically it uses pytube package to download the YouTube video and uses openai whisper to transcribe it.

For the frontend I have used streamlit which was a very good experience.

You can find the project here, hope you enjoy 

https://github.com/XamHans/video-2-text

Usecases: 
Summarize YouTube video 
Get text and used it as subtitle 
Based on the text make a blog post

**Comment by tiagojulianoferreira** [comment]
Very cool your app, congratulations!

I tried to run it but when pasting the video link and transcribing I get an HTTPError: HTTP Error 410: Gone error, but it seems to be related to pytube.

I got here because I'm looking for a way to transcribe several YouTube videos, in search of keywords in order to seek correlations between the various contents, in practice I want to identify Youtubers (Digital Influencers) of Information Technology, mapping their ideological bias, for example, between "left, right and neutral".

If anyone has any tips, they would be most welcome.

**Reply by phineas-pta** [reply]
if u have error u should raise issue in pytube repo

**Reply by tiagojulianoferreira** [reply]
There was already an issue there, I even managed to get around the error and described it there in the repo. Thanks!
https://github.com/XamHans/video-2-text/issues/1

**Reply by XamHans** [reply]
Thanks I will include it in the readme
